bases/shared/instagoric-server/server.js

```diff
@@ -11,11 +11,20 @@ import { makeSubscriptionKit } from '@agoric/notifier';
 
 const { details: X } = globalThis.assert;
 
-const CLIENT_AMOUNT =
-  process.env.CLIENT_AMOUNT || '25000000uist,25000000ibc/toyusdc';
-const DELEGATE_AMOUNT =
-  process.env.DELEGATE_AMOUNT ||
-  '75000000ubld,25000000ibc/toyatom,25000000ibc/toyellie,25000000ibc/toyusdc,25000000ibc/toyollie';
+const BASE_AMOUNT = "25000000";
+const CLIENT_AMOUNT = process.env.CLIENT_AMOUNT;
+const DELEGATE_AMOUNT = process.env.DELEGATE_AMOUNT;
+
+const COMMANDS = {
+  "SEND_BLD/IBC": "send_bld_ibc",
+  "SEND_AND_PROVISION_IST": "send_ist_and_provision",
+  "FUND_PROV_POOL": "fund_provision_pool",
+  "CUSTOM_DENOMS_LIST": "custom_denoms_list",
+};
+
+
+const PROVISIONING_POOL_ADDR = 'agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346';
```

Comment:
This address might be different for different testnets? I am surprized that it had worked so far for all of them (emerynet, devnet, other testnets)

Reply:
This a provision reserved account address regardless of any testnet. Here's a screenshot of agoric-sdk:

<img width="611" alt="Screenshot 2024-09-12 at 12 43 17 AM" src="https://github.com/user-attachments/assets/7e6ac08d-9379-4661-a59f-b2753e5dfeac">

Btw I just moved this address from client (html) to server. Was already hardcoded.

Reply:
Link:
https://github.com/Agoric/agoric-sdk/blob/07f6885c560333981d6564581b2a00735a6c428c/packages/internal/src/config.js#L51